_distro_map.yml

```diff
@@ -162,7 +168,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.14:
+    enterprise-4.15:
```

Comment:
@karanthshashank, please confirm that ROSA and OSD should start tracking 4.15 at OCP 4.15 GA.

Reply:
ACK @kalexand-rh confirm